content/articles/2018-11-parsnip-0-0-1.Rmarkdown

```diff
@@ -15,8 +15,11 @@ photo:
 library("parsnip")
 library("rstanarm")
 library("glmnet")
+# Requires development tensorflow and keras
```

Comment:
This whole first chunk will be hidden (`include = FALSE`). So, unless this is just a note to self, you might want to move it into a different chunk.

Reply:
I squash-merged this into the parsnip branch, so reviews will all be in the PR for the full post! 👍